static-instrumenter/maven-plugin/build.gradle.kts

```diff
@@ -16,13 +18,23 @@ dependencies {
   testImplementation("org.apache.maven.plugin-tools:maven-plugin-annotations:3.6.0")
   testImplementation("org.apache.maven:maven-core:3.5.0")
   testImplementation("org.slf4j:slf4j-simple")
+
+  instrumentedAgent(project(":static-instrumenter:agent-instrumenter", "shadow"))
+}
+
+task<Copy>("copyAgent") {
```

Comment:
Gradle can have problems when multiple tasks work on the same directory, in this case `processResources` and `copyAgent`. I pushed a commit to use `processResources` for this

Reply:
Thanks! Gradlu-fu is strong with you! ;)